### PR TITLE
Setup Selection Item And Selection Item Container Component Styles

### PR DESCRIPTION
**Pull request by juliannejorda**
## Notion ticket link

[Setup Selection Item and Selection Item Container component styles](https://www.notion.so/uwblueprintexecs/Setup-Selection-Item-and-Selection-Item-Container-component-styles-7007fff03b7a4e39b3875b3cb37b7aab)

## Implementation description
* Created a new custom tag with a custom theme for containers
* The custom tag is wrapped inside the container and changes colour based on state (Hover, Pressed)

## Steps to test
1. Run it on localhost:3000 and see if it changes colour depending on hover, or clicked

## What should reviewers focus on?
* See if it matches the figma prototype/design indicated on the ticket
* If the tag component changes colour depending on the current state

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: frontend/src/components/common/CustomTag.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    // aria-pressed=true;
+    if (pressed === false) {
+      setPressed(true);
+    } else {
+      setPressed(false);
+    }
+  }
+  return (
+    <Container aria-pressed={pressed} onClick={handleComponentClick}>
+      <Tag border="1px solid" borderColor="gray.700" padding="8px 12px">
```
Comment (kabiroberai) [inline frontend/src/components/common/CustomTag.tsx]: Did you intend to forward `props` here?

File: frontend/src/components/common/CustomTag.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  function handleComponentClick() {
+    // aria-pressed=true;
+    if (pressed === false) {
+      setPressed(true);
+    } else {
+      setPressed(false);
+    }
+  }
+  return (
+    <Container aria-pressed={pressed} onClick={handleComponentClick}>
```
Comment (kabiroberai) [inline frontend/src/components/common/CustomTag.tsx:30]: +1 for a11y ❤️

File: frontend/src/components/common/CustomTag.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  ...props
+}: CustomTagProps): React.ReactElement => {
+  const [pressed, setPressed] = useState(false);
+  function handleComponentClick() {
+    // aria-pressed=true;
+    if (pressed === false) {
+      setPressed(true);
+    } else {
+      setPressed(false);
+    }
```
Comment (kabiroberai) [inline frontend/src/components/common/CustomTag.tsx:27]: nit: maybe shorten this to `setPressed(!pressed)`

**Review by kabiroberai — APPROVED** [review]
LGTM! Just a couple of optional nits.

File: frontend/src/theme/containerStyles.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  sizes: {
+    md: {
+      fontSize: "md",
+    },
+    lg: {
+      fontSize: "lg",
+    },
+  },
+  variants: {
+    default: {
```
Comment (jennchenn) [inline frontend/src/theme/containerStyles.tsx:14]: do you mind making these changes in a separate variant?

**Review by jennchenn — COMMENTED** [review]
one small change otherwise looks good!!

**Review by jennchenn — APPROVED** [review]
lgtm!

**Comment by kabiroberai** [comment]
I just realized that I duplicated this work here 🙃:
https://github.com/uwblueprint/childrens-aid-society/blob/40aed618dbe6669054eb9fede128738ed9b439f5/frontend/src/components/intake/Autocomplete.tsx
